duden/word.py

```diff
@@ -53,12 +53,20 @@ def name(self):
         """
         Word without article
         """
+
+        # Find span with class "lemma__main"
+        title_element = self.soup.find('span', {"class": "lemma__main"})
+        if title_element is not None:
+            # remove soft hyphens "\xad" and return
+            return title_element.get_text().replace('\xad', '').strip() 
```

Comment:
Reusing the `clear_text` function from `common.py` would reduce code duplication.
```suggestion
            return clear_text(title_element.get_text())
```